CX-SDK-OIDC-login/src/main/java/com/cx/sdk/oidcLogin/restClient/CxServerImpl.java

```diff
@@ -114,7 +114,17 @@ public String getCxVersion() {
         HttpResponse response;
         HttpUriRequest request;
         String version;
+        HttpClientBuilder builder = HttpClientBuilder.create();
         try {
+
+            if(!isCustomProxySet(proxyParams))
```

Comment:
This code is already present in reusable method setClient. Alternatively, you could use same function like used elsewhere.